<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Испектор аэропортов</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_functions">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Поиск рейсов</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLineEdit" name="le_destination">
          <property name="text">
           <string>Любой</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Когда</string>
          </property>
          <property name="buddy">
           <cstring>le_date</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Откуда</string>
          </property>
          <property name="buddy">
           <cstring>le_departure</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="le_date"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="le_departure">
          <property name="text">
           <string>Любой</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="pb_search">
          <property name="text">
           <string>Найти</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="7">
         <widget class="QTableView" name="tv_flights"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Куда</string>
          </property>
          <property name="buddy">
           <cstring>le_destination</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Загрузка аэропорта</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="2">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Ежемесячно</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="lo_for_chart"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Ежедневно</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QGridLayout" name="lo_for_daily"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pb_show_load">
          <property name="text">
           <string>Показать</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="le_airport_name"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_stat_airport_name">
          <property name="text">
           <string>Аэропорт</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>pb_show_load</zorder>
       <zorder>tabWidget_2</zorder>
       <zorder>le_airport_name</zorder>
       <zorder>lb_stat_airport_name</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>781</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tab_functions</tabstop>
  <tabstop>le_departure</tabstop>
  <tabstop>le_destination</tabstop>
  <tabstop>le_date</tabstop>
  <tabstop>pb_search</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tv_flights</tabstop>
  <tabstop>le_airport_name</tabstop>
  <tabstop>pb_show_load</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
